tips/TIP-0045/tip-0045.md

```diff
@@ -0,0 +1,1753 @@
+  </tr>
+  <tr>
+    <td>Index</td>
+    <td>uint16</td>
+    <td>The index of the transaction input for which to claim rewards.</td>
+  </tr>
+</table>
+
+### Outputs
+
+The `Outputs` field holds the outputs that are created by the <i>Transaction Payload</i>. There are different output
+types, but they must all have an `Amount` field denoting the number of IOTA coins to deposit.
+
+### Allotments
+
+The `Allotments` field contains the list of all Mana allotments, the `Account ID`, and corresponding values that
+converts Mana provided by the inputs in the form of stored Mana in inputs or potential Mana derived from the inputs'
+IOTA coins. Mana listed in this field will be added upon commitment of the slot in which the transaction was issued, in
+form of _Block Issuance Credits_ to the account's BIC value.
+
+Note that _Block Issuance Credits_ are used to pay for the block issuance, they are burned on the slot commitment of the
+issuance slot. The good practice would be to always allot enough Mana to cover for the block issuance.
+
+### Payload
+
+The _Transaction_ itself can contain another payload as described in general in [TIP-46](../TIP-0046/tip-0046.md). The
+[semantic validity](#semantic-validation) of the encapsulating _Transaction Payload_ is unaffected by the possibly
+contained payload.
+
+### Transaction Capabilities
+
+Transaction Capabilities are represented as a `ByteArray` with a length prefix. For the transaction that contains them
+to be valid, the following conditions must hold:
+
+- The length prefix must be `0` or `1`.
+  - This condition may be relaxed in the future and implementations should already allow for greater lengths.
+- Bit indices start at `0`. For each byte, they are counted starting from the least-significant bit. If there was a
+  previous byte, the indices continue where the previous byte's indices left off.
+  - For example: In a list of two bytes with the bit patterns `0001 0000` and `0000 0100`, bits with indices `4` and
+    `10` are set.
+- There must be no trailing zero bytes in the byte array.
+  - This rule ensures that two transactions whose content is identical, and in particular, that have the same
+    capabilities will have the same binary representation.
+
+#### Capability Flags
+
+The following table shows the mapping from the bit pattern to the capability flags, where the `Flag Index` is the index
+of the bit.
+
+This list is an Allowlist: If the bit is `1` (**set**) the transaction has the capability, if the bit is `0` (**unset**)
+it does not have it.
+
+| Flag Index | Capability (if flag is set)  |
+| ---------- | ---------------------------- |
+| 0          | Can burn Native Tokens.      |
+| 1          | Can burn Mana.               |
+| 2          | Can destroy Account Outputs. |
+| 3          | Can destroy Anchor Outputs.  |
+| 4          | Can destroy Foundry Outputs. |
+| 5          | Can destroy NFT Outputs.     |
```

Comment:
I guess not but just making sure, so we need a flag for Delegation burning?

Reply:
No flag needed. The Reward Input basically acts like the flag. If it's missing, the tx fails. And since destoying a Delegation Output is a more typical operation than destroying other chain outputs, we don't add an extra flag.

Reply:
What I thought, thanks 👍🏻